README.md

```diff
@@ -59,6 +59,7 @@ To use this dbt package, you must have At least either one Fivetran **Netsuite**
 - customer
 - classification
 - department
+- employee
```

Comment:
The usage is on par with several default tables like `accounttype`  and  `accounting_books`. However it's closer to the bottom end of default usage, so we will move to use a variable configuration here.

Reply:
The `employee` object is where user access is managed. I don't think it's possible to have a configuration of NetSuite that doesn't include this table.

Reply:
Thanks for providing that context @jmongerlyra !

I think the issue we're seeing from our internal data is that customers have the `employee` table available, but do not always choose to sync it as much as the other existing tables. So we want to provide the optionality to disable this model so it's not always being synced. The variable will still be set to true so it will be available by default.